variables.tf

```diff
@@ -741,6 +733,50 @@ variable "runner_worker_docker_autoscaler_role" {
   default = {}
 }
 
+variable "runner_worker_docker_autoscaler_ingress_rules" {
+  description = "List of ingress rules for the Docker-autoscaler Runner workers"
+  type = list(object({
+    cidr_blocks      = list(string)
+    ipv6_cidr_blocks = list(string)
+    prefix_list_ids  = list(string)
+    from_port        = number
+    protocol         = string
+    security_groups  = list(string)
+    self             = bool
+    to_port          = number
+    description      = string
+  }))
+  default = []
+}
+
+variable "runner_worker_docker_autoscaler_egress_rules" {
+  description = "List of egress rules for the Docker-autoscaler Runner workers"
+  type = list(object({
+    cidr_blocks      = list(string)
+    ipv6_cidr_blocks = list(string)
+    prefix_list_ids  = list(string)
+    from_port        = number
+    protocol         = string
+    security_groups  = list(string)
+    self             = bool
+    to_port          = number
+    description      = string
+  }))
+  default = [
+    {
+      cidr_blocks      = ["0.0.0.0/0"]
+      ipv6_cidr_blocks = ["::/0"]
+      prefix_list_ids  = null
+      from_port        = 0
+      protocol         = "-1"
+      security_groups  = null
+      self             = null
+      to_port          = 0
+      description      = "Allow all egress traffic for Docker-autoscaler runner workers."
+    }
+  ]
```

Comment:
question: any specific reason for this default? As you already wrote, it might introduce a vulnerability.

suggestion: get rid of this default.

Reply:
I left it for lazy people who don't care about setting specific egresses and just want to have an ability to use the module right away. I.e. to pull docker images from Internet.
I can remove it, no problem.

Reply:
Sounds good to me, yes.